packages/fullstack/fullstack.txt

```diff
@@ -14,30 +18,40 @@ concat
 cond
 conda
 configurator
+contentful
 coworking
 datepicker
 deallocate
+deregister
 deserialize
 devkit
 directus
+dism
+dropdown
+envs
 eofline
 eprint
 eprintln
+esbenp
 fname
 focusable
 forin
 frameborder
 fullstack
 geocode
 geocoding
+geolocation
 gitignore
+globablly
```

Comment:
Do you mean `globally`? `glob ability`?

Reply:
I saw this word a few time

- On ionic 3 app.scss file
![image](https://user-images.githubusercontent.com/44493964/60500929-70fda800-9cbb-11e9-997d-d25e284a8283.png)

- [On this npm repository](https://www.npmjs.com/package/is-installed-globally)

And therefore I thought I was an English word.

Reply:
`globally` is a word. `globablly` is not.